build.gradle

```diff
@@ -55,22 +56,22 @@ bintray {
         repo = 'random'
         name = 'junit.synnefo'
         licenses = ['Apache-2.0']
-        vcsUrl  = "https://github.com/albumprinter/junit-synnefo"
+        vcsUrl = "https://github.com/albumprinter/junit-synnefo"
     }
 }
 
 dependencyManagement {
     imports {
-        mavenBom 'software.amazon.awssdk:bom:2.5.18'
+        mavenBom 'software.amazon.awssdk:bom:2.10.49'
     }
 }
 
 dependencies {
-    compile 'io.cucumber:cucumber-core:4.3.1'
-    compile 'io.cucumber:cucumber-java:4.3.1'
-    compile 'io.cucumber:cucumber-java8:4.3.1'
-    compile 'io.cucumber:cucumber-junit:4.3.1'
-    compile 'software.amazon.awssdk:s3:2.10.48'
+    compile "io.cucumber:cucumber-core:$cucumber_version"
```

Comment:
@DenUlyanov can you please use the same quotes everywhere?

Reply:
It is done